### ISSUE TITLE
Moto G6 ARCore works only when restarting

### ISSUE THREAD

**Issue by kimda90** [google-ar/arcore-unity-sdk#373]
Hi,

Using a Moto G6 - Android 8.0.0 - OPS27.82-72 with ARCore 1.4.180716093 apps like "Just A Line" only works after turning the phone off and on again.

This [post](https://forums.lenovo.com/t5/Moto-G6-Moto-G6-Plus-Moto-G6/ARCore-support-for-Moto-G6/m-p/4133823/highlight/true#M1687) in the Lenovo Forums says that ARCore 1.2.180425134 (180425134) works without the workaround.

Doing the workaround works for a while, using the previous version works but it get's updated and the cycle starts again.

Thanks

**Comment by kolboch** [comment]
First of all I am not sure if it is related. I hade some problems with acquiring camera by multiple apps on my device so what you can try is to make sure it's not used by anything else. From your post I guess you are not developing, so for your best it will be that next time you report that issue to app owner/ developers, as it might be issue with their code not specifically unity sdk. Some arcore apps sometimes are compound and contain more dependencies which might cause the issues.

**Comment by pablisho** [comment]
Hi @kimda90 Thanks for the report. I reported this internally and will be back with a response.

**Comment by sanmeet-sahaj** [comment]
I have been trying to run every example from ARCore Unity sdk  on Motorola G6, app installs successfully but none of the sample is showing output. 
AugmentedImage not detecting image.
Surface is not detected.

**Comment by sanmeet-sahaj** [comment]
Ok got to know and checked myself, things works fine only once after restarting your phone.

**Comment by kimda90** [comment]
Hi, I just wanted to know if there were any updates on this. I bought the phone because of the Google arcore support list and I really really want to start developing AR apps. 

Is there any way I can help with the debugging? I'm not sure that this is the correct repo, I'm seeing the same issues in the Android sdk repo too.

Should I be working with the Android sdk? Are you guys the same?

Thanks for all!

**Comment by fredsa** [comment]
The original issue was fixed in the 1.6.0 release.

Please feel free to file a new issue if you're still seeing issues with the latest SDK (which will make sure you also have the latest version of ARCore installed).
